src/packaging/bin/run.sh

```diff
@@ -26,12 +26,14 @@ echo ""
 echo "  HiveMQ Start Script for Linux/Unix v1.10"
 echo ""
 
+echoerr() { printf "%s\n" "$*" >&s; }
```

Comment:
have you tested this? it doesn't work for me as is, the `>&s` doesn't make sense IMO. it should be `>&2`  i think